packages/components/psammead-media-indicator/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-indicator",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "main": "dist/index.js",
```

Comment:
This package has not gone through accessibility review. We shouldn't update this to version 1.0.0 until we have done that.